### ISSUE TITLE
Binary traits problem

### ISSUE THREAD

**Issue by wangnanwei** [privefl/bigstatsr#171]
Thanks for the great work. 

I would like to implement the logistic model for binary traits using this idea. I think LDpred2 only supports linear regression currently. Maybe using summary statistics to compute the posterior mean of logistic parameters is not feasible. However, in the Lasso framework, the logistic model for binary traits can be done just with summary statistics.

Is it worthwhile to work on a logistic model for binary traits?

**Comment by privefl** [comment]
Most (if not all) polygenic score methods using GWAS summary statistics implement some linear regression.
For what I remember trying using penalized linear/logistic regression on individual-level data, it does not really make a difference.
I think I remember seeing some sumstats method published specifically implementing a logistic regression framework, but it was more complex, slower, and not particularly better.

TL;DR: do not bother
